AFB.ahk

```diff
@@ -37,9 +40,10 @@ F1:: {
 	while (IsRunning) {
 		lastID := WinExist("A")
 		robloxID := WinExist("Roblox")
-		if (robloxID) {
+		isUserPlaying := lastID == robloxID && A_TimeIdlePhysical < FIVE_MINS
+		if (robloxID && !isUserPlaying) {
```

Comment:
This is the main change here. 
@ItzNxthaniel lets test this locally for a bit to make sure this logic is sound